monitor/monitor.go

```diff
@@ -51,6 +73,19 @@ var (
 			Help:    "Latency observing CT log signed tree head (STH)",
 			Buckets: internetFacingBuckets,
 		}, []string{"uri"}),
+		certSubmitLatency: promauto.NewHistogramVec(prometheus.HistogramOpts{
+			Name:    "cert_submit_latency",
+			Help:    "Latency submitting certificate chains to CT logs",
+			Buckets: internetFacingBuckets,
+		}, []string{"uri"}),
+		certSubmitFailures: promauto.NewCounterVec(prometheus.CounterOpts{
+			Name: "cert_submit_failures",
```

Comment:
Rather than counting failures and successes separately, what about having a `cert_submit_results{status="ok"}` stat (and a `status="fail"`). That would make it clearer that you can divide the failures by the total count to get a failure percentage. It also leaves the door open to other statuses (like `canceled`) if we want to add them in the future.

Reply:
SGTM